webapp/channels/src/components/announcement_bar/configuration_bar/configuration_bar.tsx

```diff
@@ -252,16 +251,6 @@ const ConfigurationAnnouncementBar = (props: Props) => {
     }
 
     if (props.canViewSystemErrors && props.config?.SiteURL === '') {
-        let id;
-        let defaultMessage;
-        if (props.config?.EnableSignUpWithGitLab === 'true') {
-            id = t('announcement_bar.error.site_url_gitlab.full');
-            defaultMessage = 'Please configure your <linkSite>site URL</linkSite> either on the <linkConsole>System Console<linkConsole> or, if you\'re using GitLab Mattermost, in gitlab.rb.';
```

Comment:
There's a small typo I fixed here because this previously had `<linkConsole>System Console<linkConsole>` which is missing the slash on the closing tag. I don't know if it caused any issues because it's just a default value, but I wanted to mention it since it's one of the few user-visible changes here